app/controllers/api/api_controller.rb

```diff
@@ -63,6 +63,7 @@ def server_info
         :version     => Vmdb::Appliance.VERSION,
         :build       => Vmdb::Appliance.BUILD,
         :appliance   => MiqServer.my_server.name,
+        :time        => Time.now.utc,
```

Comment:
Time.now.utc.iso8601.    normalize_time(Time.now) would be preferred though that's currently a private method.

Reply:
Isn't this later passed through the normalization process anyway?

Reply:
yes, but the attribute "time" is not a time/date time declared attribute in any of the models we reference, so it won't be normalizing it as such.  (Api::Environment.time_attributes).

Reply:
Maybe we should add "time" to the list to avoid the double-pass?

Reply:
was thinking about that, also renaming this to "timestamp" but that has a different semantic.  If we want to add this to the list to leverage the normalizer, we may want to rename it first to be clear as to its content, maybe "current_time" ?